### PR TITLE
Better texts for client idle and max options

### PR DESCRIPTION
**Pull request by rmartinc**
Closes https://github.com/keycloak/keycloak/issues/9896

There were some misunderstandings with the `Client Session Idle` and `Client Session Max` (see https://github.com/keycloak/keycloak/issues/9896). It was decided to change the texts for a better description of the options.

### CODE REVIEW

**Review by mposolda — APPROVED** [review]

**Comment by rmartinc** [comment]
@andymunro Can you please take a quick look at the tooltips if you have time?

**Review by jonkoops — APPROVED** [review]

File: apps/admin-ui/public/resources/en/clients-help.json [hunk 1]
```diff
@@ -102,10 +102,10 @@
...
   "accessTokenLifespan": "Max time before an access token is expired. This value is recommended to be short relative to the SSO timeout.",
-  "clientSessionIdle": "Time a client session is allowed to be idle before it expires. Tokens are invalidated when a client session is expired. If not set it uses the standard SSO Session Idle value.",
-  "clientSessionMax": "Max time before a client session is expired. Tokens are invalidated when a client session is expired. If not set, it uses the standard SSO Session Max value.",
-  "clientOfflineSessionIdle": "Time a client offline session is allowed to be idle before it expires. Offline tokens are invalidated when a client offline session is expired. If not set it uses the Offline Session Idle value.",
-  "clientOfflineSessionMax": "Max time before a client offline session is expired. Offline tokens are invalidated when a client offline session is expired. If not set, it uses the Offline Session Max value.",
+  "clientSessionIdle": "Time a client is allowed to be idle before refreshing tokens is invalidated. The option only affects token times and not the global SSO session. If not set it uses the standard SSO Session Idle value.",
```
Comment (andymunro) [inline apps/admin-ui/public/resources/en/clients-help.json:105]:
```suggestion
  "clientSessionIdle": "Time a client is allowed to be idle before refreshing tokens is invalidated. The option only affects token times and not the global SSO session. If not set, it uses the standard SSO Session Idle value.",
```

Comment (rmartinc) [reply apps/admin-ui/public/resources/en/clients-help.json]: Thanks @andymunro! It's done now!

**Review by andymunro — APPROVED** [review]
Looks good, @rmartinc

**Pull request merged**